Problem-2.java

```diff
@@ -0,0 +1,45 @@
+Time complexity : O(n)
```

Comment:
Time complexity seems incorrect. You are searching for the index in the array. At the worst, this algorithm runs in O(n^2) time.